lib/exporters/task-metrics/summarise.js

```diff
@@ -0,0 +1,31 @@
+module.exports = (summary, task) => {
+  let {
+    taskType,
+    returnedCount,
+    wasSubmitted,
+    isOutstanding,
+    submitToActionDiff,
+    assignToActionDiff
+  } = task.metrics;
+
+  if (taskType === 'other') {
+    return summary;
+  }
+
+  if (wasSubmitted) {
+    summary[taskType].submitted++;
+  }
+
+  summary[taskType].returned += returnedCount;
```

Comment:
Is there a particular reason we count the total number of returns, but all submissions count as 1?

Reply:
Returns was specifically called out in the ticket to count every return during the report period. Submission wasn't in the original ticket but I thought it might be useful to know if the task was submitted during the period.

In my head the first submission is the important one, but maybe it's not... if you want it to count resubmissions as well it should be easy enough to add.